### ISSUE TITLE
Post 4.9.0 update, refresh token not being rotated in local storage on tokenManager.renew()

### ISSUE THREAD

**Issue by Tennyx** [okta/okta-auth-js#747]
It looks like with this fix: https://github.com/okta/okta-auth-js/pull/656

It may have broken the refresh token being replaced in local storage at authClient.tokenManager.renew(token).

New refresh tokens are coming back on /token response, but not being saved in local storage. I tried the following:

```
authClient.tokenManager.renew('accessToken')
authClient.tokenManager.renew('accessToken', 'refreshToken)
authClient.tokenManager.renew('refreshToken')
```

Reverting to 4.8.0, it seems refresh token is updating in local storage with:

`authClient.tokenManager.renew('accessToken')`

**Comment by cmacdonnacha** [comment]
Hey there, any idea when this might be fixed? I am currently reliant on version 4.9 so can't downgrade.

**Comment by aarongranick-okta** [comment]
This issue has been fixed in `okta-auth-js` version 5.2.0.
